src/components/symptom-shark/presentational/MonthReport/MonthReport.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from "react";
+import "./MonthReport.css";
+import NotesTimeline from "../NotesTimeline/NotesTimeline";
+import BulletChart from "../BulletChart/BulletChart";
+import OverallExperienceChart from "../OverallExperienceChart/OverallExperienceChart";
+import SymptomSeverityChart from "../SymptomSeverityChart/SymptomSeverityChart";
+import { getMonthName } from "../../../../helpers/date-helpers";
+import { DateRangeContext, TextBlock } from "../../../presentational";
+import { SymptomSharkVisualizationContext } from "../../container/VisualizationCoordinator/VisualizationCoordinator";
+import { startOfMonth } from "date-fns";
+
+export interface SymptomSharkMonthReportProps {
+    includeNotes: boolean;
+    includeDailyOverallFeeling: boolean;
+    productLogo?: string;
+    productName?: string;
+    productUrl?: string;
+    intervalStart?: Date;
+    innerRef?: React.Ref<HTMLTableElement>;
+}
+
+export default function (props: SymptomSharkMonthReportProps) {
+    let visualizationContext = useContext(SymptomSharkVisualizationContext);
+    if (!visualizationContext) {
+        return <TextBlock innerRef={props.innerRef}>Error: Month report must be used inside a Symptom Shark Visualization Coordinator.</TextBlock>
+    }
+
+    let dateRangeContext = useContext(DateRangeContext);
+    let intervalStart = props.intervalStart || startOfMonth(new Date());
```

Comment:
Similar comment to below.  I think this can be a one liner.

```
let intervalStart = dateRangeContext?.intervalStart ?? props.intervalStart ?? startOfMonth(new Date());
```